### DISCUSSION TITLE
Savvycan for OSX 10.12.6

### DISCUSSION THREAD

**Discussion by JacobsMess** [collin80/SavvyCAN#760] [General]
Whats the latest version of SavvyCAN that will run on OSX 10.12.6?

**Comment by collin80** [comment]
I'm honestly not sure. I would have thought that even the current version should work if you compile it yourself but I see online that QT 5.15 requires OSX 10.13 so perhaps not. SavvyCAN, as it currently sits, does require a pretty recent version of QT5, probably at least QT 5.12, maybe higher. So, maybe your chances aren't so good.

But, have you heard of OpenCore Legacy? I have an early 2013 MacBook Pro and I'm running MacOS 14.3 on it. OpenCore Legacy lets you run new MacOS versions on old hardware. So, you may be able to upgrade to a new enough version of OSX / MacOS to then use the published SavvyCAN builds.

**Comment by collin80** [comment]
I have gotten confirmation that some libraries in use require MacOS 10.15 so the answer to this question is actually: It won't work as it sits. At least the binary build requires 10.15 to operate. Maybe compiling it yourself will work on 10.13 but I can't test that.

**Reply by TessieCatEC** [reply]
Thanks Collin....  I purchase a newer MacBook Air Mac OSX 12.6.1.  The "Save" menu items don't crash the app.  HOWEVER, I can't connect to CAN-Wireless network because a dialog requires me to enter a WaP2 password.  I can't find anything on website or this forum that tells what the password is...  Can you help me sir?    Thanks so much.   steve@teamameritest.com

**Comment by collin80** [comment]
If you're talking about an ESP32 board running ESP32RET then it's a big secret what the WPA2 key is (it's aBigSecret)
